builder/image/templates/builder

```diff
@@ -112,8 +106,10 @@ if [ ! -f Dockerfile ]; then
     BUILD_OPTS+=' -v '
     BUILD_OPTS+=$(echo $CACHE_DIR)
     BUILD_OPTS+=':/tmp/cache:rw'
-    # give non-root slugbuilder user R/W perms for docker volumes
-    chown -R 2000:2000 $TMP_DIR $CACHE_DIR
+    # give slug group ownership of TMP_DIR and CACHE_DIR. This is used by the
+    # slug builder. Contents in these folders will be owned by slug:slug.
```

Comment:
`chown -R :2000 ..` seems reasonable. I didn't encounter issues with this on my Maven builds, but, the folders would then be created in slugbuilders /app folder and, since they weren't part of the git repo, there wouldn't be any problem.